app/form/aboutthetradinghistory/CaravansOpenAllYearForm.scala

```diff
@@ -32,9 +32,9 @@ object CaravansOpenAllYearForm {
   val caravansOpenAllYearForm: Form[(AnswersYesNo, Option[Int])] =
     Form(
       tuple(
-        "openAllYear"  -> createYesNoType("error.caravans.openAllYear.required"),
-        "weeksPerYear" -> mandatoryIfEqual(
-          "openAllYear",
+        "areCaravansOpenAllYear" -> createYesNoType("error.caravans.openAllYear.required"),
+        "weeksPerYear"           -> mandatoryIfEqual(
+          "areCaravansOpenAllYear",
```

Comment:
If you changing field id you need also to change it on CYA page
https://github.com/hmrc/tenure-cost-and-trade-records-frontend/blob/main/app/views/answers/answersAboutYourTradingHistory.scala.html#L528-L536